src/TLuaInterpreter.cpp

```diff
@@ -546,52 +548,50 @@ int TLuaInterpreter::resetProfile(lua_State* L)
 // Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#selectString
 int TLuaInterpreter::selectString(lua_State* L)
 {
-    Host& host = getHostFromLua(L);
-
-    int s = 1;
-    QString windowName; // only for 3 argument case, will be null if not assigned to which is different from being empty
+    int s = 0;
+    QString windowName = cMainConsoleName;
     if (lua_gettop(L) > 2) {
-        if (!lua_isstring(L, s)) {
-            lua_pushfstring(L, R"(selectString: bad argument #%d type (window name as string, is optional {defaults to "main" if omitted}, got %s!))", s, luaL_typename(L, s));
-            lua_error(L);
-            return 1;
-        } else {
-            // We cannot yet properly handle non-ASCII windows names but we will eventually!
-            windowName = QString::fromUtf8(lua_tostring(L, s));
-            if (windowName == QLatin1String("main")) {
-                // This matches the identifier for the main window - so make it
-                // appear so by emptying it...
-                windowName.clear();
-            }
-            s++;
+        if (!lua_isstring(L, ++s)) {
+            lua_pushfstring(L,
+                            R"(selectString: bad argument #%d (window name as string is optional {omit for \"%s\" console}, got %s!)",
+                            s,
+                            cMainConsoleName.toUtf8().constData(),
```

Comment:
Unwise - since we are abstracting it to a `const`, it is still better coding IMHO to keep everything tied to that constant so that **should** we ever change it we then do not have to track down *every* single place where it *hasn't* been abstracted...